test/utils.ts

```diff
@@ -33,6 +33,7 @@ function loadInteraction(name: interactionNames) {
 export function configurationToExecute() {
   const excelInteraction = loadInteraction("Excel");
   const openDocumentInteraction = loadInteraction("OpenDocument");
+  const LibreOfficeInteraction = loadInteraction("LibreOffice");
```

Comment:
Actually,  ODS and FODS are both open document, the difference is that fods is a Flat Open Document spreadhseet, basically an uncompressed ODS

Reply:
I renamed them